modyn/selector/internal/selector_strategies/downsampling_strategies/empty_downsampling_strategy.py

```diff
@@ -0,0 +1,32 @@
+from modyn.selector.internal.selector_strategies.downsampling_strategies import AbstractDownsamplingStrategy
+
+
+class EmptyDownsamplingStrategy(AbstractDownsamplingStrategy):
```

Comment:
This class is a little bit hacky but I think that it'ss better to abstract as much as possible and "fix" in this way